File: Number2.java
package algorithm;
import java.util.Scanner;
public class Number2 {

	public static void main(String[] args) {
		// 2. 자연수의 합
		// 자연수 A, B가 주어지면 A부터 B까지의 합을 수식과 함께 출력하세요.
				
				Scanner stdIn = new Scanner(System.in);
				System.out.print("A의 수를 입력하시오:");
				int a = stdIn.nextInt();
				System.out.print("B의 수를 입력하시오:");
				int b = stdIn.nextInt();
				int sum = 0;
				for(int i = a; i<b; i++) {
					System.out.print(i+" + ");
					sum+= i;
				}
				sum+= b;
				System.out.print(b+" = "+ sum);
			}

}

File: number2.cpp
#include <stdio.h>

// 자연수의  합  
// 자연수 A, B가 주어지면 A부터 B가지의 합을 수식과 합께 출력하시오. 
 
int main() {
	int a;
	int b;
	int sum=0;
	scanf("%d %d", &a, &b);
	for(int i=a; i<b; i++)
	{
		printf("%d + ",i);
		sum+=i;		 		
	}
	sum+= b;
	printf("%d = %d", b, sum);
	
	return 0;
}
